app/assets/javascripts/admin/addon/models/color-scheme.js

```diff
@@ -88,6 +88,28 @@ export default class ColorScheme extends EmberObject {
     return [`"${this.name}": {`, buffer.join(",\n"), "}"].join("\n");
   }
 
+  schemeObject() {
+    const extractColors = (property) =>
+      Object.fromEntries(
+        this.colors.map((color) => [color.get("name"), color.get(property)])
+      );
+    return {
+      dark: extractColors("dark_hex"),
+      light: extractColors("hex"),
+    };
+  }
+
+  /**
+   * @returns a JSON representation of the color scheme.
+   */
+  dump() {
+    return (
+      JSON.stringify(this.name) +
```

Comment:
The string concatenation used here is safe, but the previous one was not. Users may use double quotes `"` or backslashes `\` in the name, and not using stringify will break the JSON export.

Reply:
Yes, I thought so too!